graphql/testdata/task1.graphql

```diff
@@ -0,0 +1,10 @@
+{
+  task(taskId: "wuzzup") {
+    logs {
+      AllLogLink
+      TaskLogLink
+      AgentLogLink
+      SystemLogLink
```

Comment:
why are these capitalized? i think convention around schema suggests all things should be camel cased